python/semantic_kernel/core_plugins/crew_ai/crew_ai_models.py

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+from enum import Enum
+from typing import Any
+
+from semantic_kernel.kernel_pydantic import KernelBaseModel
+
+
+class CrewAIEnterpriseKickoffState(str, Enum):
+    """The Crew.AI Enterprise kickoff state."""
+
+    Pending = "PENDING"
+    Started = "STARTED"
+    Running = "RUNNING"
+    Success = "SUCCESS"
+    Failed = "FAILED"
+    Failure = "FAILURE"
+    Not_Found = "NOT FOUND"
+
+
+class CrewAIStatusResponse(KernelBaseModel):
+    """Represents the status response from Crew AI."""
+
+    state: CrewAIEnterpriseKickoffState
+    result: str | None
+    last_step: dict[str, Any] | None
+
+
+class CrewAIKickoffResponse(KernelBaseModel):
+    """Represents the kickoff response from Crew AI."""
+
+    kickoff_id: str
+
+
+class CrewAIRequiredInputs(KernelBaseModel):
+    """Represents the required inputs for Crew AI."""
+
+    inputs: dict[str, str]
+
+
+class InputMetadata(KernelBaseModel):
```

Comment:
the more I see this the more I want it to have a name with CrewAI in it, alternative would be to do a translation from a kernelParameterMetadata object to this...